app/components/Data.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import React, { Component } from "react";
+import { Content } from "native-base";
+import { Query } from "react-apollo";
+
+import { Pending } from "components";
+
+type Props = {
+  presentData: any => any,
```

Comment:
It would be nice to have this be more specific. Could be worth playing around with on a separate branch. It feels like this is enough to keep Flow happy, but won't actually give us any of the benefits of type checking.